pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -705,37 +708,67 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 		switch t := v.(type) {
 		case *extensions.Deployment:
 			//delete deployment
-			rpDeployment, err := kubectl.ReaperFor(extensions.Kind("Deployment"), client)
+			label := labels.SelectorFromSet(labels.Set(map[string]string{selector: t.Name}))
```

Comment:
@kadel If we put
 ```golang 
label := labels.SelectorFromSet(labels.Set(map[string]string{selector: t.Name}))
```
outside `switch` and `for` how do we access the name of the object we want to delete created, here we are accessing it using `t.Name`.